### PR TITLE
FFM-8095: FF archiving

### PR DESCRIPTION
**Pull request by StrawberryPad**
[**Preview link**](https://64afe15fa18c940c53b66166--harness-developer.netlify.app/docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag#archive-and-restore-flags)

### CODE REVIEW

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md [hunk 1]
```diff
@@ -23,53 +23,96 @@ After you have created a flag, you can edit the following details:
...
+You might want to check if you have stale flags so that you can decide whether to archive them. In Harness, flags are counted as potentially stale if in the past 60 days:
 
-When you are finished with a Feature Flag, it's best practice to remove it to keep your Flags and application organised and tidy. This topic describes how to delete a Feature Flag on the Harness Platform and using [Git](/docs/feature-flags/manage-featureflags-in-git-repos).
+* They haven't been changed or evaluated.
+* Their default rules or target rules haven't been added to, or updated.
+* They haven't been toggled on or off.
+
+If a flag gets stale, and then any of the above occurs, the flag is no longer considered stale.
+
+**To view a list of potentially stale flags:**
```
Comment (brian-f-harness) [inline docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:51]:
```suggestion
To view a list of potentially stale flags:
```

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md [hunk 2]
```diff
@@ -23,53 +23,96 @@ After you have created a flag, you can edit the following details:
...
+### Archive a flag
+
+You can archive a flag if it's not a prerequisite to other flags. If it is, remove it as a prerequisite, and then archive it.
+
+:::info notes
+1. When you archive a flag, it is removed from all environments.
+2. You cannot archive a flag if it is a prerequisite to other flags. Remove the flag as a prerequisite, and then you can archive it.
+:::
+
+**To archive a flag:**
```
Comment (brian-f-harness) [inline docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:71]:
```suggestion
To archive a flag:
```

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md [hunk 3]
```diff
@@ -23,53 +23,96 @@ After you have created a flag, you can edit the following details:
...
+
+Restoring a flag makes it available for evaluation if the flag is referenced in your code. That is, you haven't removed the references to it when you archived the flag, or you've reinstated those references.
 
 :::info note
- Make sure you are ready to delete the Flag from all of your Environments. When you delete a Flag on the Harness Platform or on Git, it is removed from all Environments.
+When you delete a flag, it is permanently removed from all environments.
 :::
 
-### Delete a flag using the Harness Platform
+**To restore a flag:**
```
Comment (brian-f-harness) [inline docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:93]:
```suggestion
To restore a flag:
```

File: docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md [hunk 4]
```diff
@@ -23,53 +23,96 @@ After you have created a flag, you can edit the following details:
...
+1. Next to the flag you want to restore, select **more options** (**︙**), and then select **Restore**.
+1. On the confirmation screen, select **Restore**.
 
-3. In **Delete Flag**, click **Delete**.
+### Delete an archived flag
 
+Archived flags are automatically deleted 30 days after you archive them. But if you're sure you want to delete the flag immediately, you can delete an archived flag before that 30-day period ends.
 
-### Delete a flag using Git
+**To delete an archived flag:**
```
Comment (brian-f-harness) [inline docs/feature-flags/ff-creating-flag/edit-and-delete-a-feature-flag.md:106]:
```suggestion
To delete an archived flag:
```

**Review by brian-f-harness — DISMISSED** [review]
lgtm. Left a couple minor suggestions.

**Comment by brian-f-harness** [comment]
You may want to consider adding a brief intro to break up the headings at the beginning of the topic.

**Review by KimberlyFields — APPROVED** [review]

**Comment by KimberlyFields** [comment]
Merging per request from Nick Smyth.

**Pull request merged**